.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: ROSA Scheduled Create cluster
     runs-on: ubuntu-latest
     steps:
-      - run: gh workflow run -R keycloak/keycloak-benchmark rosa-cluster-create.yml
+      - run: gh workflow run -R keycloak/keycloak-benchmark rosa-multi-az-cluster-create.yml
```

Comment:
When we no longer want a daily Multi-AZ setup we can just update the worfklow being called.